loguru.cpp

```diff
@@ -1311,7 +1311,7 @@ namespace loguru
 		if (custom_level_name) {
 			snprintf(level_buff, sizeof(level_buff) - 1, "%s", custom_level_name);
 		} else {
-			snprintf(level_buff, sizeof(level_buff) - 1, "% 4d", verbosity);
+			snprintf(level_buff, sizeof(level_buff) - 1, "% 4d", (int8_t)verbosity);
```

Comment:
Maybe we could be more explicit and do `static_cast<int8_t>(verbosity)`?

Reply:
Yeah, sure, why not. Just keep in mind that all of this was a slightly hackish solution to begin with.